knowledge-base/kb-telerik-wpf-controls-build-platform.md

```diff
@@ -0,0 +1,35 @@
+---
+title: WPF Telerik controls are not displayed correctly in x64 build platform
+description: In some version of Visual Studio the designer does no display WPF Telerik controls correctly when the build platform is x64
+type: troubleshooting
+page_title: Setting the x64 build mode, the designer does not display correctly the controls
+slug: kb-telerik-wpf-controls-build-platform
+position: 
```

Comment:
Set a position. I am not sure if the empty position won't break something in our implementations. I am using position: 0.